src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -274,35 +276,12 @@ export default {
 			this.$refs.fileUploadInput.click()
 		},
 
-		/**
-		 * Uploads and shares the selected files
-		 * @param {object} event the file input event object
-		 */
-		async processFiles(event) {
-			// Store the token in a variable to prevent changes when changing conversation
-			// when the upload is still running
-			const token = this.token
+		handleFileInput(event) {
+			const files = Object.values(event.target.files)
 			// Create a unique id for the upload operation
 			const uploadId = new Date().getTime()
```

Comment:
Since you always create it in the same way, handle it inside `processFiles` ?

Reply:
I'd like to have the ID at hand within the components for future features

Reply:
It's the only way we can access the file and do things with them e.g. keep track of upload status